lib/models/group.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:reaxit/models.dart';
+
+part 'group.g.dart';
+
+enum MemberGroupType { committee, society, board }
```

Comment:
I am not sure what this name means? It reads like the type of a member group, but its the type of the group

Reply:
The values of the enum should give enough context